.circleci/config.yml

```diff
@@ -3,12 +3,14 @@ jobs:
   build:
     machine:
       image: ubuntu-1604:201903-01
+    environment:
+      RUST_VERSION: nightly-2019-07-14
     steps:
       - checkout
       - restore_cache:
           keys:
-            - v1-cargo-global-cache-{{ arch }}-{{ checksum "Cargo.lock" }}
-            - v1-cargo-global-cache-
+            - v1-cargo-global-cache-{{ arch }}-{{ checksum "Cargo.lock" }}-{{ checksum ".circleci/RUST_VERSION" }}
```

Comment:
is it possible to have this using environment variable `$RUST_VERSION` to avoid duplicated nightly version?

Reply:
unfortunately not, I tried that

Reply:
also not possible to load env from file?